android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -470,6 +471,13 @@ class GestureHandlerOrchestrator(
     return found
   }
 
+  private fun shouldHandlerSkipHoverEvents(handler: GestureHandler<*>, action: Int): Boolean {
+    val shouldSkipEvents =
+      handler !is HoverGestureHandler && handler !is RNGestureHandlerRootHelper.RootViewGestureHandler
+
+    return action in listOf(MotionEvent.ACTION_HOVER_EXIT, MotionEvent.ACTION_HOVER_ENTER, MotionEvent.ACTION_HOVER_MOVE) && shouldSkipEvents
```

Comment:
```suggestion
    return shouldSkipEvents && action in listOf(MotionEvent.ACTION_HOVER_EXIT, MotionEvent.ACTION_HOVER_ENTER, MotionEvent.ACTION_HOVER_MOVE)
```

Imo, it's easier to read this way

Reply:
I agree, done in [c1a9158](https://github.com/software-mansion/react-native-gesture-handler/pull/3462/commits/c1a91588266e9604bcc22ccd3f5b220383cf51a2)